typings/index.d.ts

```diff
@@ -115,9 +115,8 @@ declare module 'klasa-dashboard-hooks' {
 		sslOptions?: SecureContextOptions;
 	}
 
-	export interface DashboardClientOptions extends KlasaClientOptions {
-		dashboardHooks?: KlasaDashboardHooksOptions;
-	}
+	// Types are inherited from augmentation
```

Comment:
These seem redundant now?

Reply:
It's left there since the [typedef exists](https://github.com/dirigeants/klasa-dashboard-hooks/blob/4c38f102c6dbc01001ee56254dc33dfc503964dd/src/lib/Client.js#L19-L22).